installer-builder/darwin/scripts/postinstall

```diff
@@ -25,4 +25,11 @@ sudo rm -rf "/opt/finch/"
 sudo rm -rf "/private/var/run/finch-lima"
 sudo rm -rf "/private/etc/sudoers.d/finch-lima"
 
+# Forget previous pkgutil packages starting with org.Finch except for the current version
+echo "Remove historical pkgutil packages..."
```

Comment:
Since this is in the postinstall script, will it remove the version of Finch that was just installed from the registry too? Or does that only get added to the registry after the postinstall script?

Reply:
grep -v '^org\.Finch\.__VERSION__' excluded the current version, so it will only remove historical versions